package.json

```diff
@@ -44,6 +44,7 @@
     "@babel/preset-env": "7.4.5",
     "@babel/preset-typescript": "7.3.3",
     "@babel/register": "7.4.4",
+    "@graphql-tools/delegate": "^6.0.10",
```

Comment:
`graphql-tools` was broken down into many packages. Longer term, we need to update all the parts of `graphql-tools`, but for now we can incrementally update particular parts.